internal/service/endpoints.go

```diff
@@ -38,6 +38,14 @@ type GatheringRulesResponse struct {
 	Rules   interface{} `json:"rules"`
 }
 
+// RemoteConfigurationResponse structure represents HTTP response with
+// remote configuration content
+type RemoteConfigurationResponse struct {
```

Comment:
isn't this a duplicate of `RemoteConfiguration` from the repository package?

Reply:
yep good catch. THanks